alerting/provider/gotify/gotify.go

```diff
@@ -0,0 +1,105 @@
+package gotify
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/TwiN/gatus/v5/alerting/alert"
+	"github.com/TwiN/gatus/v5/client"
+	"github.com/TwiN/gatus/v5/core"
+)
+
+const DefaultPriority = 5
+
+// AlertProvider is the configuration necessary for sending an alert using Gotify
+type AlertProvider struct {
+	// ServerURL is the URL of the Gotify server
+	ServerURL string `yaml:"server-url"`
+
+	// AppToken is the token of the application to send messages to
+	AppToken string `yaml:"app-token"`
```

Comment:
Any chance you could rename this to just token?

Reply:
Hi @TwiN ,

Done.